tests/docker/cloudInit.sh

```diff
@@ -0,0 +1,40 @@
+#
+#
+# note ...
+# must run as " . ./script.sh"
+# that is, if you don't start with a dot then the
+# environment export does not work.
```

Comment:
Is this comment intended for users running this script directly?
It looks like the only place this script is uses here doesn't use the source (dot) notation.

Reply:
Removed comment from file.